x/confidentialtransfers/keeper/msg_server.go

```diff
@@ -242,6 +254,14 @@ func (m msgServer) Withdraw(goCtx context.Context, req *types.MsgWithdraw) (*typ
 		return nil, sdkerrors.Wrapf(sdkerrors.ErrInsufficientFunds, "insufficient funds to withdraw %s %s", req.Amount, req.Denom)
 	}
 
+	gasSoFar := ctx.GasMeter().GasConsumed()
+	multiplier := m.Keeper.GetRangeProofGasMultiplier(ctx)
+
+	// Consume additional gas according to the multiplier as range proofs are computationally expensive.
+	if multiplier > 1 {
```

Comment:
curious, is there a chance it could trigger with existing validation in place?

Reply:
Hmm not sure what you mean here? Default multiplier is 10 so it should always trigger, but 1 is also a valid multiplier (meaning don't multiply anything), meaning it will skip the following step